app/module-types/node.js

```diff
@@ -35,9 +38,20 @@ exports.NodeModule = class {
 
   start () {
     if (this.script) {
+      const moduleLogPath = join(logPath, this.name)
+      if (!fs.existsSync(moduleLogPath)) {
+        fs.mkdirSync(moduleLogPath)
```

Comment:
So you're creating a directory `/log`, and inside it there's a directory for each module, and inside that there's a file for that module? If you're not saving a few files, it's not worth a directory

Reply:
I am saving a few files- all the rotated files for each module. When I used a lower interval it quickly got hard to find the file I'm looking for. With five modules and the main process, after two rotations or the length of your average competition you'll have 18 different log files, without sorting them into sub directories it got impossible to find the right log.